evasquez26/angular4-startup
# S3-ONE-110 Upload profile picture

## Story / Task

see [ONE-110](https://bixlabs.atlassian.net/browse/ONE-110)


## What this PR does?

Adds functionality to update a user's profile picture

### How to test / reproduce

- Login to the app
- Go to My profile
- Click Edit
- Select a picture
- Click Save
- You should see the new profile picture in the profile bar and in header

### TODOs
- [x] Rebase after merging #32 

### Related PRs
N/A

## Migrations
NO


Thank you
